perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -104,7 +104,7 @@ public Group createGroup(PerunSession sess, Group parentGroup, Group group) thro
 
 			//We catch those exceptions, but they should never be thrown, so we just log them.
 		} catch (WrongAttributeValueException | WrongReferenceAttributeValueException e) {
-			log.info("Exception thrown in createGroup method, while it shouldn't be thrown. Cause:{}",e);
+			log.debug("Exception thrown in createGroup method, while it shouldn't be thrown. Cause:{}",e);
```

Comment:
If something shouldn't be there at all, it's not the debug level. It has to be the error level.